routes/basket.py

```diff
@@ -0,0 +1,43 @@
+from fastapi import APIRouter, Depends
+from fastapi.responses import JSONResponse
+from bson import ObjectId
+
+from database.database import Basket
+from models.basket import BasketBase
+from dependency import get_current_user
+from serializers.basket import basket_serializer
+from utils import validator
+
+router = APIRouter(prefix="/basket")
+
+# Create Basket -> POST /api/basket
+@router.post("/")
+async def create_basket(basket: BasketBase, user: dict = Depends(get_current_user)):
+
+    basketInserted = await Basket.insert_one(basket.dict())
+    if basketInserted:
+        return JSONResponse(status_code=200, content={"basketId": str(basketInserted.inserted_id)})
+
+    return JSONResponse(status_code=400, content={"error": "Basket could not be created"})
+
+# Get Basket By Id -> GET /api/basket/{id}
+@router.get("/{id}")
+async def get_basket_by_id(id: str, user: dict = Depends(get_current_user)):
+    basket = await Basket.find_one({"_id": ObjectId(id)})
```

Comment:
We can search the basket of the user using the UserId from the get_current_user as user and basket will have one-to-one relationship
So no need to send a param in the request url